hash_practice/exercises.py

```diff
@@ -1,19 +1,54 @@
 
+import collections
+from typing import Collection
+
+
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n)
+        Space Complexity: O(n)
     """

```

Comment:
👍 Correct if the words are limited in size.  If the words are not limited in size the time complexity is O(n * m log m) where m is the length of the word.